src/applications/vaos/services/patient/index.js

```diff
@@ -242,12 +278,21 @@ export async function fetchFlowEligibilityAndClinics({
   directSchedulingEnabled,
   featurePastVisitMHFilter = false,
   isCerner = false,
+  removeFacilityConfigCheck = false,
   featureUseBrowserTimezone,
 }) {
+  // Helper to avoid confusion
+  const keepFacilityConfigCheck = !removeFacilityConfigCheck;
+  // When the removeFacilityConfigCheck flipper is removed, all directSchedulingAvailable should be changed to directSchedulingEnabled
   const directSchedulingAvailable =
-    locationSupportsDirectScheduling(location, typeOfCare) &&
+    (locationSupportsDirectScheduling(location, typeOfCare) ||
+      removeFacilityConfigCheck) &&
     directSchedulingEnabled;
 
+  const typeOfCareRequiresPastHistory =
+    typeOfCare.id !== TYPE_OF_CARE_IDS.MENTAL_HEALTH_SERVICES_ID ||
```

Comment:
Primary care also requires past history check. Was there a reason you took out this check? 
`typeOfCare.id !== TYPE_OF_CARE_IDS.PRIMARY_CARE`